dbt_meshify/storage/dbt_project_editors.py

```diff
@@ -48,7 +48,7 @@ def load_resource_yml(
         self, path: Path, name: str, resource_type: NodeType, nested_name: Optional[str] = None
     ) -> Dict:
         """Load the Model patch YAML for a given ModelNode."""
-        raw_yml = YAMLFileManager.read_file(path)
+        raw_yml = YAMLFileManager.read_file(path) if path.exists() else {}
```

Comment:
Great change! I suspect an alternative approach would be to handle the `FileNotFoundError`, but this seems a bit more elegant since we know to check for the path first.